pyproject.toml

```diff
@@ -1,12 +1,95 @@
 [build-system]
-requires = [
-    "setuptools >= 66",
-    "setuptools_scm >= 7.0.0",
-    "wheel >= 0.38.4",
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "seagap"
+dynamic = ["version"]
+description = "Community Seafloor Global Navigation Satellite Systems - Acoustic (GNSS-A) Positioning Software"
+readme = "README.md"
+license = "MIT"
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "fsspec",
```

Comment:
What are the latest recommendations on having versions/version ranges specified? I know I've been bitten many times by inadvertently using different package versions between the team or deployments.

Reply:
Usually we try not to pin to always get the latest, but I totally agree that divergence in package versions can occur. So I'll pin now. Thanks for the suggestion!